hvac_services/demo/sale_order_line.xml

```diff
@@ -0,0 +1,238 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo auto_sequence="1" noupdate="1">
+    <record id="sale_order_line_38" model="sale.order.line">
+        <field name="order_id" ref="sale_order_11"/>
+        <field name="product_id" ref="product_product_repair"/>
+        <field name="analytic_distribution" eval="{'2': 100}"/>
```

Comment:
2 is related to an id from the db. I'm not sure you need to define the distribution, but if you do, you'll need use a ref instead of an hardcoded id